integration-tests/tests/src/tests/sync/flexible.ts

```diff
@@ -396,6 +399,88 @@ describe.skipIf(environment.missingServer, "Flexible sync", function () {
     });
   });
 
+  describe("Errors", () => {
```

Comment:
We might want to make this description more specific since we're testing errors in other suites as well 👍 
```suggestion
  describe("Compensating writes", () => {
```